ReadMe.md

```diff
@@ -0,0 +1,99 @@
+# LibWalter v0.1
+
+LibWalter is a code collection which aims to fill in the gaps in the BeOS API, released under the MIT license. Designed to be compatible with BeOS R5, Zeta 1.x, and Haiku.
+
+## 0. Index:
+
+1. Included classes and usage notes
+2. How to use LibWalter
+3. How to contribute
+
+## 1. Included classes and usage notes
+
+#### AutoTextControl
```

Comment:
When the heading of this section is on level 2 (`##`) the subsection should be level 3 (`###`), not remain on level 4. Applies to the whole document.